MediaBrowser.Providers/Plugins/Tmdb/Configuration/PluginConfiguration.cs

```diff
@@ -26,5 +27,45 @@ public class PluginConfiguration : BasePluginConfiguration
         /// Gets or sets a value indicating the maximum number of cast members to fetch for an item.
         /// </summary>
         public int MaxCastMembers { get; set; } = 15;
+
+        /// <summary>
+        /// Gets or sets a value indicating the poster image size to fetch.
+        /// </summary>
+        public string? PosterSize { get; set; }
+
+        /// <summary>
+        /// Gets or sets the available options for poster size.
+        /// </summary>
+        public List<string> PosterSizeOptions { get; set; } = new List<string>();
```

Comment:
This is not really part of the configuration. You're overwriting them every time you read the config from TMDb. A better approach is an XHR call to an endpoint that spits out the valid size options.